GPU/Common/PresentationCommon.cpp

```diff
@@ -726,6 +719,13 @@ void PresentationCommon::CopyToOutput(OutputFlags flags, int uvRotation, float u
 		}
 	}
 
+	// Finally, we compensate the y vertex positions for the backbuffer for any flipping.
+	if ((flags & OutputFlags::POSITION_FLIPPED) || (flags & OutputFlags::BACKBUFFER_FLIPPED)) {
+		for (int i = 0; i < 4; i++) {
+			verts[i].y = -verts[i].y;
```

Comment:
Does this work even when the display layout is not centered?  What about rotation (`In this mode, we ignore the g_display_rot_matrix.  Apply manually.`)?

-[Unknown]

Reply:
Yes, it does work with non centered layouts.

As for rotation, it would also work but we only use that type of rotation on Vulkan, which doesn't need any flipping here.